policy_definitions/network_security/policy_set_definition_qby_NetworkSecurity.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "QBY-NetworkSecurity",
+    "type": "Microsoft.Authorization/policySetDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "description": "Enforce namingconform NSG to have a rule that denys all inbound traffic in Vnet. Ensures all subnets and NSG to be namingconform and with their Vnet in name associated",
+        "displayName": "NetworkSecurity",
+        "metadata": {
+            "version": "0.1.0",
+            "category": "NetworkSecurity"
+        },
+        "parameters": {
+        },
+        "policyDefinitions":  [ 
+            {"policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/QBY-Deny-NSG-DenyAllInboundTraffic"},
+            {"policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/QBY-Deny-Vnet-Subnet-NSG"}
```

Comment:
You can use "policyDefinitionReferenceId" to give the policy a nice readable name instead of just the resource id. See for example [Update Management](https://github.com/qbeyond/terraform-azurerm-archetype-lib/blob/dfd5e13ad27520f748de0d7ada099235b8699416/policy_definitions/update_management/policy_set_definition_qby_deploy_update_management.tmpl.json#L50)